examples/ACUnit/ACUnit.ino

```diff
@@ -0,0 +1,102 @@
+/*
+ * Example for how to use Lock device as garage door opener
```

Comment:
Example for _Lock device as garage door opener_ ?

Reply:
Changed